### ISSUE TITLE
Make our doc site theme-able

### ISSUE THREAD

**Issue by puru-hk** [SAP/fundamental-ngx#3127]
Fundamental-ngx website has hardcoded colors and colors linked to theming parameters. That's why some parts in HCB are not visible.
The goal is convert all the colors touse theming parameters

Fundamental core component: "Pagination"

Description: The items in the ‘playground’ section is not visible in the High contrast Black theme.

Expected: All the UI elements should get adapt to the High contrast Black theme when the theme is changed.

Attached screenshot:

In Fiori 3 theme

![image](https://user-images.githubusercontent.com/32538291/91557559-4efcc580-e952-11ea-887f-d60327b9d1e7.png)

In High Contrast Black theme

![image](https://user-images.githubusercontent.com/32538291/91557590-5c19b480-e952-11ea-9e37-86ae0b6e8b02.png)
